lib/models/survey_info.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:object_mapper/object_mapper.dart';
+
+class SurveyInfo with Mappable {
+  String? title;
+  String? description;
+  String? coverImageUrl;
+
+  @override
+  void mapping(Mapper map) {
+    map<String>("title", title, (v) => title = v as String?);
+    map<String>("description", description, (v) => description = v as String?);
+    map<String>(
+        "cover_image_url", coverImageUrl, (v) => coverImageUrl = v as String?);
+  }
```

Comment:
this could be added to the [Backend] ticket too

Reply:
@sleepylee into [this](https://github.com/markgravity/flutter-survey/pull/82)